dss/storage/bundles.py

```diff
@@ -90,6 +85,8 @@ def save_bundle_manifest(replica: Replica, uuid: str, version: str, bundle: dict
     data = json.dumps(bundle).encode("utf-8")
     fqid = BundleFQID(uuid, version).to_key()
     created, idempotent = idempotent_save(handle, replica.bucket, fqid, data)
+    if should_cache_file(content_type='application/json', size=len(data)):
```

Comment:
Files should have been pre-cached already in PUT /file, so we don't need to do it again here.

Reply:
Removed.